public/scripts/app.js

```diff
@@ -1,7 +1,174 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
+   //initialize all modals           
+   $('.modal').modal();
+   $('select').material_select();
+
+  $.ajax({
+    method: 'GET',
+    url: '/api/shops',
+    success: renderAllShops
+  });
+
+
+
+
+
+
+  $('#shop-form form').on('submit', function(e) {
+    e.preventDefault();
+    var formData = $(this).serialize();
+    console.log('formData', formData);
+    $.post('/api/shops', formData, function(shop) {
+      console.log('shop after POST', shop);
+      renderShop(shop);  //render the server's response
+    });
+    $(this).trigger("reset");
+  });
+
+  // catch and handle the click on an add song button
+   	$('#modal1').on('click', '#submit', handleAddShopClick);
+	$('#submit').on('click', handleNewShopSubmit);
+	$('#shops').on('click','.delete-shop', handleDeleteShopClick);
+
+
 
-// your code
 
 });
+
+
+
+function renderAllShops (shops) {
+	
+	shops.forEach(function (shop) {
+		console.log(shops)
+		renderShop(shop);
+	});
+}
+
+
+
+
+
+
+
+
+
+
+// render shop function
+function renderShop (shop) {
+	console.log('Rending these damn pet shops', shop)
+
+var shopsHTML = (`
+	 	  <div class="card">
+            <div class="card-image waves-effect waves-block waves-light">
+              <img class="activator" src="${shop.image}">
+            </div>
+            <div class="card-content">
+            <span class="card-title activator grey-text text-darken-4">${shop.shopName}<i class="material-icons right">${shop.careTakerType}</i></span>
+              <p><a href="#">${shop.website}</a></p>
+
+            </div>
+            <div class="card-reveal">
+              <span class="card-title grey-text text-darken-4">${shop.address}<i class="material-icons right">${shop.phoneNumber}</i></span>
+              <p>Here is some more information about this product that is only revealed once clicked on.</p>
+            </div>
+              <div class='card-footer hidden' >
+              <button class='btn btn-info edit-shop'>Edit Shop</button>
+              <button class='btn btn-danger delete-shop'>Delete</button>
+            </div>
+          </div>`);
+$('#shops').prepend(shopsHTML)
+ 		
+};
+
+
```

Comment:
Lots of spaces... are these needed?

Reply:
No they're not Carlynn. Jevell tighten this code up!